<launch>
    <node pkg="map_conversion" type="local_environment_node" name="local_environment" output="screen" />
    <node pkg="map_conversion" type="global_submap_node" name="global_submap" output="screen" />

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find map_conversion)/config/rviz_test.rviz" />
    </group>
</launch>
